src/Canvas.tsx

```diff
@@ -44,6 +49,8 @@ export interface CanvasContainerProps extends CanvasProps {
 
   /**
    * Whether the canvas is pannable or not.
+   *
+   * TODO What does "pannable" means in Reaflow context? I don't understand it
```

Comment:
Remove comment - it means 'can pan' which is scrollbars. There is a demo of this turned off.